src/avfoundation.cs

```diff
@@ -1077,15 +1358,27 @@ interface AVAudioMixerNode : AVAudioMixing {
 	interface AVAudioOutputNode {
 
 	}	
+	
+	// AudioBufferList is binded as AudioBuffers which is present in AudioToolbox which is not present on the watch. bug filled: 
+	// https://bugzilla.xamarin.com/show_bug.cgi?id=59145
+#if !WATCH && XAMCORE_2_0
+	[Watch (4,0), TV (11,0), Mac (10,10), iOS (8,0)]
+	delegate AudioBuffers AVAudioIONodeInputBlock (uint frameCount);
+#endif
 
-	[NoTV]
-	[NoWatch]
-	[iOS (8,0)][Mac (10,10)]
-	[BaseType (typeof (AVAudioIONode))]
+ 	[Watch (4,0)]
+ 	[iOS (8,0)][Mac (10,10)][TV (11,0)]
+ 	[BaseType (typeof (AVAudioIONode))]
 	[DisableDefaultCtor] // returned Handle is nil
 	// note: sample source (header) suggest it comes from AVAudioEngine properties
 	interface AVAudioInputNode : AVAudioMixing {
 
+		// uncomment once bug 59145 is fixed.
+#if !WATCH && XAMCORE_2_0
+		[Watch (4,0), TV (11,0), Mac (10,13), iOS (11,0)]
+		[Export ("setManualRenderingInputPCMFormat:inputBlock:")]
+		bool SetManualRenderingInputPCMFormat (AVAudioFormat format, AVAudioIONodeInputBlock block);
```

Comment:
that should be `Pcm`, not `PCM`

Reply:
Ok, but we have an issue with some of the constants:

		[Field ("AVNumberOfChannelsKey")]
		NSString AVNumberOfChannelsKey { get; }
		
		[Field ("AVLinearPCMBitDepthKey")]
		NSString AVLinearPCMBitDepthKey { get; }
		
		[Field ("AVLinearPCMIsBigEndianKey")]
		NSString AVLinearPCMIsBigEndianKey { get; }
		
		[Field ("AVLinearPCMIsFloatKey")]
		NSString AVLinearPCMIsFloatKey { get; }
		
		[Field ("AVLinearPCMIsNonInterleaved")]
		NSString AVLinearPCMIsNonInterleaved { get; }

This were added in hash 5830166c6, should we add a bug and create a PR with the new names?